mage_ai/data_preparation/models/block/sql/__init__.py

```diff
@@ -279,6 +280,50 @@ def execute_sql_code(
                             verbose=False,
                         ),
                     ]
+    elif DataSource.DUCKDB.value == data_provider:
+        from mage_ai.io.duckdb import DuckDB
+
+        loader = DuckDB.with_config(config_file_loader)
+        schema = schema or 'main'
```

Comment:
Does user need to enter this schema in the io_config.yaml?

Reply:
Make sense! I thought user enter schema at Mage UI Block header. It is also a good idea to add it into io_config. I created a `DUCKDB_SCHEMA` in io_config also tested it in the Mage UI.